src/components/postDetail/ReviewList/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+
+import classNames from 'classnames/bind';
+
+import { REVIEW_SORT_OPTIONS } from '@/constants';
+import { getReviewPageSize } from '@/utils/getPageSize';
+
+import Dropdown from '@/components/commons/Dropdown';
+import Pagination from '@/components/commons/Pagination';
+import ReviewCard from '@/components/commons/ReviewCard';
+import ReviewSummary from '@/components/postDetail/ReviewSummary';
+import { REVIEW_LIST_DATA } from '@/constants/mockData/reviewList';
+import { useDeviceType } from '@/hooks/useDeviceType';
+import usePaginatedDataList from '@/hooks/usePaginatedDataList';
+import useSortedDataList from '@/hooks/useSortedDataList';
+
+import { Order, Review, ReviewResponse, SortOption } from '@/types';
+
+import styles from './ReviewList.module.scss';
+
+const cx = classNames.bind(styles);
+
+type ReviewListProps = {
+  list?: ReviewResponse;
+};
+
+const ReviewList = ({ list = REVIEW_LIST_DATA }: ReviewListProps) => {
+  const [page, setPage] = useState(1);
+
+  const currentDeviceType = useDeviceType();
+  const pageSize = getReviewPageSize(currentDeviceType);
+
+  const initialDataList: Review[] = list.reviews;
+  const initialSortOption: SortOption<Review> = {
+    key: 'rating',
+    order: 'desc',
+    type: 'number',
+  };
+
+  const [sortOption, setSortOption] = useState(initialSortOption);
+
+  const sortedDataList = useSortedDataList({
+    initialDataList,
+    sortOption,
+  });
+
+  const pagedDataList = usePaginatedDataList({
+    initialDataList: sortedDataList,
+    page,
+    setPage,
+    postsPerPage: pageSize,
+  });
+
+  const handleOptionChange = (value: string | number) => {
+    setSortOption({ key: 'rating', order: value as Order, type: 'number' });
```

Comment:
```suggestion
    setSortOption((prev) => ({...prev, order: value as Order}));
```

value 외의 나머지는 동일하니까 이렇게 작성해도 좋을거같습니다!